staycation-ts/src/components/accomodation-card/AccomodationCard.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import './AccomodationCard.css';
+
+const AccomodationCard: React.FC = () => {
+    const sugarSpiceApartments = require('../../assets/images/sugar_spice_apartments.png');
+    const startImage = require('../../assets/images/star.png');
+    const accommodation: {
+        title: string,
+        location: string,
+        price: number,
+        categorization: number
+    } = {
+        "title": "Sugar & Spice Apartments",
+        "location": "Split",
+        "price": 75,
+        "categorization": 3
+    };
+
+    const categorizationRender = (categorization: number) => {
+        let content = [];
+
+        for(let i = 0; i < categorization; i++) {
+            content.push(<img src={ startImage } />);
```

Comment:
Budući da ove slike zvjezdica stavljamo u array, kad renderiramo taj array, moramo imati **key** za svaki element kako bi ih se moglo razlikovati. Trenutno ovo javlja error u consoli kad se otvori Dev Tools.
Da bi to popravio, dovoljno je dodati key na sljedeći način:
`content.push(<img src={ startImage } key={i} />);`